syntax = "proto3";

message UdpHello {
    int32 port = 1;
}
